src/components/ui.js

```diff
@@ -156,6 +156,7 @@ AFRAME.registerComponent('ui', {
       }
       case name === 'clear': {
         if (!this.pressedObjects[name]) {
+          this.toggleCaptureCamera();
```

Comment:
Need to get another button in the menu for this.  If people like the change maybe @feiss can hook up a camera button.